src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/Benefit.java

```diff
@@ -56,7 +56,7 @@ public enum Benefit {
     THIRTY_HOURS_FREE_CHILDCARE("30 Hours Free Childcare", "Gofal Plant am ddim - 30 awr", "058", "thirtyHoursFreeChildcare", List.of("058"), false, DwpAddressLookupService::thirtyHoursFreeChildcareOfficeMappings, AirLookupService::getJsaBereavementBenefitVenue, SSCS5),
     GUARANTEED_MINIMUM_PENSION("Guaranteed Minimum Pension (COEG)", "Isafswm Pensiwn Gwarantedig", "034", "guaranteedMinimumPension", List.of("034"), false, DwpAddressLookupService::guaranteedMinimumPensionOfficeMappings, AirLookupService::getJsaBereavementBenefitVenue, SSCS5),
     NATIONAL_INSURANCE_CREDITS("National Insurance Credits", "Credydau Yswiriant Gwladol", "030", "nationalInsuranceCredits", List.of("030"), false, DwpAddressLookupService::nationalInsuranceCreditsOfficeMappings, AirLookupService::getJsaBereavementBenefitVenue, SSCS5),
-    INFECTED_BLOOD_APPEAL("Infected Blood Appeal", "Apêl Gwaed Heintiedig", "093", "infectedBloodAppeal", List.of("093"), false, DwpAddressLookupService::infectedBloodAppealMappings, AirLookupService::getInfectedBloodCompensationVenue, SSCS8);
+    INFECTED_BLOOD_APPEAL("Infected Blood Compensation", "Apêl Gwaed Heintiedig", "093", "infectedBloodAppeal", List.of("093"), false, DwpAddressLookupService::infectedBloodAppealMappings, AirLookupService::getInfectedBloodCompensationVenue, SSCS8);
```

Comment:
should the enum also be renamed INFECTED_BLOOD_APPEAL ?

Reply:
probably in a future task of replacing all instances of appeal with compensation in relation to this across the tribs and sya repo's (same with the shortName code as infectedBloodAppeal) as clean up